global.json

```diff
@@ -1,5 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.100"
+    "version": "7.0.100",
+    "rollForward": "latestFeature"
```

Comment:
I ran into issues with `rollForward` in combination with some of the source generators. I have since moved away from the problematic ones, so I think this should be safe now.